### ISSUE TITLE
Bundling global dependencies issue.

### ISSUE THREAD

**Issue by VasilioRuzanni** [systemjs/builder#5]
@guybedford 

The system.js is super-convenient due to its ability to load any module and automatically resolve its format. Awesome.

When system.js loads different modules as different files which is pretty good for development - everything works perfect and no other configuration is even required aside from setting the `paths` for third-party dependencies. All things "just work".

There are problems though when trying to "bundle" everything for optimized production usage. Issue raised when moved the medium-sized Angular app to using ES6 with Traceur and ES6 Module Loader and tried to prepare the combined/concat-ted build file.

I would like to use SystemJS bundled in production too, so that I don't have to deal with AMD/etc at all. Calling `System.import(...)` works just fine for me and nowadays seems to be recommended approach according to [your article](http://guybedford.com/practical-workflows-for-es6-modules) (Dynamic Workflow 4).

Now, **the issue**.
While in dev, if I refer to dependency like

``` JavaScript
import angular from 'angular';

// The `angular` is what it should be here, things work and everyone is happy.
angular.module(...);
```

When I use `systemjs-builder` to create a bundle though (my app imports the `angular` and that gets into the bundle too, which is OK), the result of `import` looks like

``` JavaScript
import angular from 'angular';

// angular == { angular: Object, ng339: 3 }
```

For me, it seems like `systemjs-builder` might have something like `shim` config missing (like we do for RequireJS). After intensive googling, no ways to provide any kind of shim config for build were found. Maybe there is some other way to configure that for builder? Or am I missing something?

So, basically the questions are:
- How `systemjs-builder` is different from just SystemJS loader and why it changes the format my module is being imported? Is this expected behavior? - SystemJS seems to somehow know in development what is getting exported from "global" module.
- How to configure that? Some `systemjs-builder` tweak or some configuration option?
- Any pointers to some other options on how to solve this?

Thanks in advance.
